model/validation.go

```
@@ -838,6 +838,55 @@ func ValidateParentAndDrain(drainTime, parentShutdown *duration.Duration) (errs
 				parentShutdownDuration.Seconds(), drainDuration.Seconds()))
 	}
 
+	if drainDuration > (time.Hour * 1) {
+		errs = multierror.Append(errs,
+			fmt.Errorf("Istio drain time %v must be less than 1 hour", drainDuration.Minutes()))
+	}
+
+	if parentShutdownDuration > (time.Hour * 1) {
+		errs = multierror.Append(errs,
+			fmt.Errorf("Istio parent shutdown time %v must be less than 1 hour",
+				parentShutdownDuration.Minutes()))
+	}
+
+	return
+}
+
+// ValidateRefreshDelay validates the discovery refresh delay time
+func ValidateRefreshDelay(refresh *duration.Duration) (errs error) {
+	if err := ValidateDuration(refresh); err != nil {
+		errs = multierror.Append(errs, err)
```

You can return the error directly here since `errs` should be empty.